source/conf.py

```diff
@@ -344,3 +344,37 @@
 # If true, figures, tables and code-blocks are automatically numbered if they
 # have a caption. At same time, the numref role is enabled.
 numfig = True
+
+# -- Custom configuration -------------------------------------------------
+
+from jinja2 import Environment, StrictUndefined
+
+# Use Jinja to parse RST files first
+def rstjinja(app, docname, source):
+    """
+    Render our pages as a jinja template for fancy templating goodness.
+    """
+    jinja_env = Environment(undefined=StrictUndefined)
```

Comment:
We are barely using Jinja2 here. Just for these version strings. I think its appropriate. Though presumably in the future we might want to set different strings for all the Jinja2 delimiters in case we add tutorials on building simple OOD apps using Python.

Though its one find/replace in files, so switching to this later will take no time at all. As long as we document the escape hatch. We could wait till we start adding documentation about app development that includes code with handlebars or mustache or jinja.